src/js/components/AppVersionComponent.jsx

```diff
@@ -10,36 +10,58 @@ var UNSPECIFIED_NODE =
     }
   });
 /*eslint-enable react/display-name */
+
+function invalidateValue(value, suffix) {
+  if (value == null) {
+    return (
+      <UNSPECIFIED_NODE />
+    );
+  } else {
+    return (
+      <dd>{value} {suffix}</dd>
+    );
+  }
+}
```

Comment:
Could simplify this a bit, since you're only returning one tag, you could just:
```
return <UNSPECIFIED_NODE />;
```
and
```
return <dd>{value} {suffix}</dd>;
```

Don't need the parentheses AFAIK